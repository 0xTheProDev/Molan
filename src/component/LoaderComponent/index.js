// @flow
import React from 'react';
import { Dimmer, Loader, Image } from 'semantic-ui-react';
import _img from './text.png';
import Styles from './index.module.css';

function LoaderComponent() {
  return (
    <div className={Styles['loader-container']}>
      <Dimmer active inverted>
        <Loader inverted content="Loading" />
      </Dimmer>
      <Image src={_img} />
    </div>
  );
}

export default LoaderComponent;
