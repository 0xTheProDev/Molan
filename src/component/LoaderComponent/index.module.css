.loader-container {
    height: 200px !important;
}
