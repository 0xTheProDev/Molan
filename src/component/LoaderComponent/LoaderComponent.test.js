import React from 'react';

import { LoaderComponent } from './LoaderComponent';

describe('Loader Component', () => {

  it('should render', () => {
    const wrapper = shallow(<LoaderComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
