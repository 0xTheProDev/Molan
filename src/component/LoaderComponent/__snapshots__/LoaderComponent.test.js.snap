// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should render 1`] = `
<div
  className="loader-container"
>
  <Dimmer
    active={true}
    inverted={true}
  >
    <Loader
      content="Loading"
      inverted={true}
    />
  </Dimmer>
  <Image
    as="img"
    src="text.png"
    ui={true}
  />
</div>
`;
